Untitled.ipynb

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Delete this file

Reply:
Definitely, just using it to draft right now.